test_apps/smokeTests/gridTools.cpp

```diff
@@ -79,13 +79,17 @@ void MakeTriangle(Grid *grid, float minVal, float maxVal)
         for (size_t j = 0; j < y; j++) {
             for (size_t i = 0; i < x; i++) {
                 value = value == minVal ? maxVal : minVal;
-                grid->SetValueIJK(i, j, k, value);
+                if (addRandomMissingValues) {
+                    if (rand()%2) grid->SetValueIJK(i,j,k, grid->GetMissingValue());
```

Comment:
Using rand() is considered a pretty bad programming practice (https://stackoverflow.com/questions/52869166/why-is-the-use-of-rand-considered-bad). Please consider using its better alternatives.